.github/workflows/ci.yml

```diff
@@ -12,6 +12,12 @@ jobs:
     if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
 
     runs-on: ${{ matrix.os }}
+
+    services:
+      minio:
+        image: lazybit/minio:latest
+        ports:
+          - 9000:9000
```

Comment:
This breaks all but Ubuntu it seems:
```console
Error: Container operations are only supported on Linux runners
```

Maybe we need dedicated Linux only CI for s3 support today?

Reply:
yes, will do